### ISSUE TITLE
Switch from PhantomJS to Electron

### ISSUE THREAD

**Issue by bchr02** [johntitus/node-horseman#50]
@johntitus I'm not sure if you have seen, but [segmentio/nightmare](https://github.com/segmentio/nightmare) has now moved over from PhantomJS to [Electron](http://electron.atom.io/). Is this something that you are considering? 

The nightmare [README.md](https://github.com/segmentio/nightmare/blob/master/Readme.md) claims that it's faster than PhantomJS:
> Under the covers it uses Electron, which is similar to PhantomJS but faster and more modern.

**Comment by johntitus** [comment]
I'm not, at the moment. Electron is very unstable, and you can't run it truly headless in linux.

https://github.com/segmentio/nightmare/issues/224#issuecomment-141575361

That said, if/when electron becomes more stable and if/when they can run headlessly, then people can probably just use [segmentio/nightmare](https://github.com/segmentio/nightmare). The two api's are very similar.

I would be interested in seeing a benchmark on speed between the two.

**Comment by bchr02** [comment]
It seems that nightmare caters to a different type of user than your repo. For example, they don't have built in jQuery support and it appears they are not willing to support newly generated windows (see https://github.com/segmentio/nightmare/issues/148). I say "not willing" because @reinpk closed this Issue instead of leaving it open. His comment here (https://github.com/segmentio/nightmare/issues/148#issuecomment-139900939) states that he doesn't believe the feature is currently possible: 
> i don't believe this is possible in electron, so i'm going to close this as not possible unfortunately. if you have an example of a popup script would be great to test it out though

But doesn't explain why. And when I wrote a suggestion as to how it might be done (https://github.com/segmentio/nightmare/issues/148#issuecomment-139958006) I have yet to get a reply:
> @reinpk BrowserWindow.getAllWindows() returns an array of all the BrowserWindow object references. Surely this could be used to incorporate some sort of interaction with newly generated windows.

You are probably right though on the stability. I have used Atom editor which is built on Electron and it has been unstable to say the least.
